src/components/elements/checkbox.js

```diff
@@ -35,7 +35,6 @@ const StyledCheckbox = styled.div`
         if (props.secondary && props.checked) return '2px solid var(--color-red)'
         return '2px solid var(--color-grey-5)'
     }};
-    border-radius: 3px;
```

Comment:
we are using this element over the website, would you please change the `border-radius` inside the inheritance component if it's needed?

Reply:
`border-radius` has been removed from checkbox style throughout the project.